package.json

```diff
@@ -27,10 +27,13 @@
     "prebuild": "npm run doc-listing-inject",
     "build": "node scripts/build.js",
     "postbuild": "rm -rf examples/node_modules && mkdir examples/node_modules && cp -r node_modules/@webcomponents node_modules/requirejs examples/node_modules/",
-    "clean": "rm -rf node_modules package-lock.json elements/*/{node_modules,package-lock.json,_temp} test/vrt-snapshots",
+    "clean": "rm -rf node_modules elements/*/{node_modules,package-lock.json}",
```

Comment:
It'd be cool if you had a "clean:all" command which would run all the separate clean commands at one time.

Reply:
Oh I like that - adding that now

Reply:
added